dnf-plugins-core.spec

```diff
@@ -74,18 +102,168 @@ BuildRequires:  python3-sphinx
 Requires:       python3-dnf >= %{dnf_lowest_compatible}
 Requires:       python3-dnf < %{dnf_not_compatible}
 Requires:       python3-hawkey >= %{hawkey_version}
+%if %{undefined rhel}
+Requires:       python3-dnf-plugins-extras-copr
+%endif
 Conflicts:      %{name} <= 0.1.5
 # let the both python plugin versions be updated simultaneously
 Conflicts:      python2-%{name} < %{version}-%{release}
 Conflicts:      python-%{name} < %{version}-%{release}
+Provides:       python3-dnf-plugins-extras-repoclosure = %{epoch}:%{version}-%{release}
+Provides:       python3-dnf-plugins-extras-repograph = %{epoch}:%{version}-%{release}
+Provides:       python3-dnf-plugins-extras-repomanage = %{epoch}:%{version}-%{release}
+Obsoletes:      python3-dnf-plugins-extras-repoclosure < %{dnf_plugins_extra}
+Obsoletes:      python3-dnf-plugins-extras-repograph < %{dnf_plugins_extra}
+Obsoletes:      python3-dnf-plugins-extras-repomanage < %{dnf_plugins_extra}
 
 %description -n python3-%{name}
-Core Plugins for DNF. This package enhances DNF with builddep, config-manager,
-copr, debuginfo-install, download, needs-restarting, repoquery and
-reposync commands. Additionally provides generate_completion_cache and noroot
-passive plugins.
+Core Plugins for DNF. This package enhances DNF with builddep, config-manager, debuginfo-install,
+download, needs-restarting, repoclosure, repograph, repomanage and reposync commands. Additionally
+provides generate_completion_cache and noroot passive plugins.
 %endif
 
+%package -n python2-dnf-plugin-leaves
```

Comment:
oh I see, show-leaves is alternative to yum-plugin-show-leaves. So is dnf-plugin-leaves alternative to yum-plugin-remove-with-leaves then?